src/types/components.d.ts

```diff
@@ -313,7 +313,7 @@ export interface PageProps {
 export class Page extends Component<PageProps> {}
 
 export interface ParagraphProps {
-  size?: "base" | "xs" | "sm" | "md" | "lg";
+  size?: "xs" | "sm" | "base" | "lg" | "xl";
```

Comment:
When you change Props please make sure, that the ones you removed are no longer used in the project. 
Often the compiler will tell you this. 
There is a `size="md"` used in the `NewsDetail`-Component. Please change the property there.